LLama/LLamaContext.cs

```diff
@@ -75,6 +76,7 @@ public LLamaContext(LLamaWeights model, IContextParams @params, ILogger? logger
 
             @params.ToLlamaContextParams(out var lparams);
             NativeHandle = SafeLLamaContextHandle.Create(model.NativeHandle, lparams);
+            _emptyState = GetState();
```

Comment:
Could you check the size of this state? Generally you don't want to be keeping states around in memory if you don't need to, they can be huge. The empty state might be a special case exception to that rule though.

Reply:
Removed it altogether, as it's not needed